cmd/manager/main.go

```diff
@@ -81,20 +81,17 @@ func main() {
 	printVersion()
 
 	operatorNamespace, err := k8sutil.GetOperatorNamespace()
-	if err != nil {
+	if err != nil && err != k8sutil.ErrRunLocal {
```

Comment:
This was preventing me from running locally since ErrRunLocal is returned in that case. If this change is invalid, let me know.